rxjava1/src/main/java/io/micronaut/rxjava1/instrument/RxJava1TracingInstrumentation.java

```diff
@@ -76,10 +76,10 @@ public Action0 call(Action0 action0) {
             if (instrumenter != null) {
                 return () -> {
                     try {
-                        instrumenter.beforeInvocation();
+                        //instrumenter.beforeInvocation();
```

Comment:
Maybe remove instead of commenting out?

Reply:
Removing will break the instrumentation. This PR is not ready